src/components/sidebar/components/nav-list/index.tsx

```diff
@@ -0,0 +1,6 @@
+import { ReactNode } from 'react'
+import s from '../../sidebar.module.css'
+
+export default function NavList({ children }: { children: ReactNode }) {
+	return <ul className={s.navList}>{children}</ul>
```

Comment:
Extracted this into a reusable component since I needed some of the sidebar styles in the custom sidebar for the bookmarks view and this seemed better than importing from the stylesheet.